app/views/core_induction_programmes/years/show.html.erb

```diff
@@ -0,0 +1,44 @@
+<%= govuk_breadcrumbs breadcrumbs: course_year_breadcrumbs(current_user, @course_year) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">Your module materials</h1>
+
+    <% if @cip.course_year_two_id %>
+      <%= render SubnavComponent.new do |component| %>
+        <%= component.nav_item(path: year_path(@cip.course_year_one_id)) do %>
+          Year 1
+        <% end %>
+        <%= component.nav_item(path: year_path(@cip.course_year_two_id)) do %>
+          Year 2
+        <% end %>
+      <% end %>
+    <% end %>
+
+    <% if current_user&.admin? || policy(@course_year).edit? %>
+      <div class="govuk-button-group">
+        <% if current_user&.admin? %>
+          <%= govuk_link_to "Create CIP Module", cip_create_module_path(@cip), button: true %>
+          <%= govuk_link_to "Create CIP Lesson", cip_create_lesson_path(@cip), button: true %>
+        <% end %>
+        <% if policy(@course_year).edit? %>
+          <%= govuk_link_to "Edit year content", edit_year_path(@course_year), button: true %>
+        <% end %>
```

Comment:
I think for time being we can lump it all behind a check 'is user an admin?', they are the only ones who can edit content for now.